tests/unit/test_column_noise.py

```diff
@@ -749,22 +860,48 @@ def test_generate_phonetic_errors(dummy_dataset, column):
         NOISE_TYPES.make_phonetic_errors.name
     ]
     noised_data = NOISE_TYPES.make_phonetic_errors(
-        dummy_dataset, config, RANDOMNESS0, "dataset", column
+        data, config, RANDOMNESS0, "dataset", column
     )
     data = data.squeeze()
 
     # Validate we do not change any missing data
-    missing_mask = data.isna()
-    assert noised_data[missing_mask].isna().all()
+    missing_mask = data == ""
```

Comment:
Was this change necessary?

Reply:
Yes there were no nan in the series but empty strings.